docs/io-use.md

```diff
@@ -105,21 +106,21 @@ configs:
    tableName: "test_jdbc"
 ```
 
-## Get available connectors
+## Prepare a connector
 
 Before starting using connectors, you can perform the following operations:
 
-* [Reload connectors](#reload)
+* [Reload connectors](#reload-connectors)
 
-* [Get a list of available connectors](#get-available-connectors)
+* [Get a list of available connectors](#get-a-list-of-available-connectors)
 
-### `reload`
+### Reload connectors
```

Comment:
```suggestion
### `reload`
```
Keep the original text as it is since it's the same as the command https://pulsar.apache.org/reference/#/next/pulsar-admin/sources?id=reload

Besides, 2-level headings already show the explanations

<img width="292" alt="image" src="https://github.com/apache/pulsar-site/assets/50226895/045e24e0-592a-4fa1-86c4-ebd12e8a3a2e">